lib/routes/components.js

```diff
@@ -85,6 +85,12 @@ module.exports = app => {
 			if (component.type === 'module' && component.resources.demos) {
 				demos = await app.repoData.listDemos(component.repo, component.id);
 
+				demos.forEach(async demo => {
+					if (demo.display.html) {
+						demo.display.plainHTML = await fetchHTML(demo.supportingUrls.html);
```

Comment:
I think this can be achieved by just using `request-promise-native`, which is used by the Repo Data API client. Then you could use:

```
const request = require('request-promise-native');
// ...
demo.display.plainHTML = await request(demo.supportingUrls.html);
```
with no need for the new lib code